Modules/Loadable/Markups/Resources/UI/qSlicerMarkupsModule.ui

```diff
@@ -1070,6 +1070,47 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <widget class="ctkCollapsibleButton" name="angleModeCollapsibleButton">
+     <property name="text">
+      <string>Angle Settings</string>
+     </property>
+     <property name="checked">
+      <bool>false</bool>
+     </property>
+     <layout class="QGridLayout" name="gridLayout_2">
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>Angle mode:</string>
+        </property>
+       </widget>
+      </item>
+      <item row="0" column="1">
+       <widget class="QComboBox" name="angleModeComboBox">
```

Comment:
If we add angle mode to the GUI then we must allow the user to set the rotation axis, too. You can use a ctkMatrixWidget.